python.py

```diff
@@ -0,0 +1,20 @@
+# here is a coment
```

Comment:
Typo inside comment in word `comentary`. Possible corrections: commentary (7), coventry (0),